web-client/src/presenter/actions/TrialSessionWorkingCopy/getTrialSessionWorkingCopyCaseNotesFlagAction.js

```diff
@@ -0,0 +1,13 @@
+import { state } from 'cerebral';
+
+/**
+ * getTrialSessionWorkingCopyCaseNotesFlagAction
+ *
+ * @param {object} providers.get the cerebral get function
+ * @returns {Object} caseNotesFlag
+ */
+
+export const getTrialSessionWorkingCopyCaseNotesFlagAction = ({ get }) => {
+  const caseNotesFlag = get(state.modal.caseNotesFlag);
+  return { caseNotesFlag };
```

Comment:
Nit: is there a more accurate, descriptive name for this? looks like it's being used for showing case notes on the printable session copy so like, `includeCaseNotes` or `showCaseNotes` maybe?

Reply:
Renamed to showCaseNotes throughout all files.